build-support/bin/generate_faas_complete_platforms.py

```diff
@@ -1,16 +1,29 @@
 # Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+# NB: This script requires that the underlying docker installation
+# has `buildx` support, and also that the underlying system has
+# QEMU installed so that Docker can run images of a different
+# architecture.
+
+# This script can be run via `pants run`, but it needs to be run
+# with the `--no-watch-filesystem` flag, because the contents of the
+# `awslambda` backend depend on the resources generated by this script.
+# If run without the `--no-watch-filesystem` flag, the script will
+# loop infinitely, because it restarts itself whenever the output files
+# change.
```

Comment:
Huh, this is surprising to me. This file isn't marked as `restartable=True`, which would imply to me that it _should not_ be being restarted https://www.pantsbuild.org/2.22/reference/targets/python_source#restartable

https://github.com/pantsbuild/pants/blob/81a1cdb699f80b38c0b0d759620568cd03156b95/build-support/bin/BUILD#L7-L9

Looks like a bug! https://github.com/pantsbuild/pants/issues/21243

```suggestion
# If run without the `--no-watch-filesystem` flag, the script will
# loop infinitely, because it restarts itself whenever the output files
# change (FIXME #21243).